sdk/samples/iot/iot_sample_common.c

```diff
@@ -231,6 +230,7 @@ void iot_sample_read_environment_variables(
     switch (name)
     {
       case PAHO_IOT_PROVISIONING_SAMPLE:
+      case PAHO_IOT_PNP_WITH_PROVISIONING_SAMPLE:
```

Comment:
nit: move before PAHO_IOT_PROVISIONING_SAMPLE to keep ordering consistent (alphabetical)

Reply:
fixed.